Formula/lefthook.rb

```diff
@@ -1,8 +1,8 @@
 class Lefthook < Formula
   desc "Fast and powerful Git hooks manager for any type of projects"
   homepage "https://github.com/evilmartians/lefthook"
-  url "https://github.com/evilmartians/lefthook/archive/refs/tags/v0.7.7.tar.gz"
-  sha256 "4653b79a1d5cc4675da579f885726b4b6b6379777744723f84f45296d5e04c14"
+  url "https://github.com/evilmartians/lefthook/archive/refs/tags/v0.8.0.tar.gz"
+  sha256 "4873473f6a9dcbcb7ca756aa254b0bec8576435917a92be7d5081abf5a496065"
```

Comment:
Hey, lefthook maintainer is here.

I have found that v0.8.0 tag was made against wrong commit (this one: https://github.com/evilmartians/lefthook/commit/87f84cf93f21b12939061010841239a158d69f03) instead of directly following it commit that bumped version number (this one: https://github.com/evilmartians/lefthook/commit/f0283528ca1a1db3671b0d941735a5219546bc09).

I have force-pushed tag to point to correct commit, so new SHA256 sum should be used instead, here it is:

```suggestion
  sha256 "32b9107dfa12ac28560720509d44c0bc173e6092f5095a6b1a2787859ea2ce9b"
```

Check lefthook 0.8.0 release for details: https://github.com/evilmartians/lefthook/releases/tag/v0.8.0

Reply:
Please don't retag releases. The [git manual](https://git-scm.com/docs/git-tag#_on_re_tagging) says re-tagging is "the insane thing" to do, and as you noticed here it's hard to deal with for package managers.

Reply:
Ok, gotcha. I won't retag it back :sweat_smile: